NMRangeSliderIOS.js

```diff
@@ -16,10 +16,20 @@ const NMRangeSliderIOS = React.createClass({
     lowerCenter: React.PropTypes.object, // CGPoint?
     upperCenter: React.PropTypes.object, // CGPoint?
     onChange: React.PropTypes.func,
+    thickness: React.PropTypes.number,
```

Comment:
What do you think of calling it 'trackThickness'? Then it is more clear what thickness it affects.